sites/public/page_content/locale_overrides/bn.json

```diff
@@ -1,7 +1,7 @@
 {
   "welcome": {
     "title": "মধ্যে সাশ্রয়ী মূল্যের আবাসন খুঁজুন",
-    "heroText": "ডেট্রয়েট হোম কানেক্ট হল এমন একটি জায়গা যেখানে আপনি আপনার গৃহস্থালি চাহিদা এবং বৈশিষ্ট্যের উপর ভিত্তি করে বাসস্থান খুঁজে পেতে পারেন।"
+    "heroText": "ডেট্রয়েট হোম কানেক্ট হল এমন একটি জায়গা যেখানে আপনি আবাসন খুঁজে পেতে পারেন যা আপনি আপনার পরিবারের প্রয়োজনের ভিত্তিতে সামর্থ্য করতে পারেন।"
```

Comment:
It looks like you specifically didn't translate "Detroit Home Connect" for the other languages, but you did here?

Reply:
I did my best guess based on Google translate here, hopefully it looks OK.